app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PostsController < ApplicationController
   def index
-    @posts = Post.where(author_id: params[:user_id])
+    @posts = Post.includes(:author).where(author_id: params[:user_id])
     @user = User.includes(:posts).find(params[:user_id])
```

Comment:
- [ ] Great job here. I can see that you are fetching users together with their posts which is a good thing and a requirement for this project, but we need to do more here. Instead of just getting posts, we also need comments for those posts. I have attached a screenshot of this requirement for your reference. (we can do all these at line 4)

![posts](https://user-images.githubusercontent.com/53124548/235899487-1f11d6a0-b6ac-45fe-b760-e92894aef2ff.png)